setup.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2022-2024, The ORBIT Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Installation script for the 'orbit.ext_template' python package."""
+
+import os
+import toml
+
+from setuptools import setup
+
+# Obtain the extension data from the extension.toml file
+EXTENSION_PATH = os.path.dirname(os.path.realpath(__file__))
+# Read the extension.toml file
+EXTENSION_TOML_DATA = toml.load(os.path.join(EXTENSION_PATH, "config", "extension.toml"))
+
+# Minimum dependencies required prior to installation
+INSTALL_REQUIRES = [
+    # NOTE: Add dependencies
+    # "numpy",
+    # "torch==2.0.1",
+]
+
+# Installation operation
+setup(
+    name=EXTENSION_TOML_DATA["python"]["module"][0]["name"],
```

Comment:
Not sure about the `name` and `packages` arguments. Using `EXTENSION_TOML_DATA["python"]["module"][0]["name"]` feels wrong, but otherwise the user also has to configure `setup.py`. Any ideas?

Reply:
Yeah this they can fill up once.

Reply:
Sounds good. This is updated now.